modules/highgui/include/opencv2/highgui.hpp

```diff
@@ -401,6 +401,18 @@ CV_EXPORTS_W void imshow(const String& winname, InputArray mat);
  */
 CV_EXPORTS_W void resizeWindow(const String& winname, int width, int height);
 
+/** @brief Resizes window to the specified size
```

Comment:
It's better to use `/** @overload` command here instead of useless copy of description.